spec/models/article_spec.rb

```diff
@@ -43,4 +43,14 @@
     expect(Tweet.first.latitude).to eq(nil)
     expect(Tweet.first.article_id).to eq(Article.first.id)
   end
+
+  it "can generate keywords" do
+    VCR.use_cassette("nytimes_articles") do
+      allow(Article).to receive(:_get_latlon).and_return({"lat" => 1.5, "lng" => 1.5})
```

Comment:
Line is too long. [86/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.